provider/resource_playbook.go

```diff
@@ -498,23 +642,39 @@ func resourcePlaybookUpdate(data *schema.ResourceData, meta interface{}) error {
 		)
```

Comment:
Can't comment on the line above, but `runAnsiblePlayErr` is just the exit code. We should be giving the users `runAnsiblePlayOut`, instead. This is what will have the useful playbook output.

Reply:
fixed.